app/models/journal_manager.rb

```diff
@@ -355,7 +355,7 @@ def self.with_send_notifications(send_notifications, &block)
     self.send_notification = send_notifications
 
     result = block.call
-
+  ensure
     self.send_notification = old_value
 
     result
```

Comment:
FYI this is a noop as the result will be the last expression before the ensure no matter what you do in ensure.
So one could just change the line before ensure to:

```
block.call
```

Reply:
Didn't know that. Thanks for explaining @machisuji.